docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -51,6 +51,75 @@ broker:
     enable: false
 ```
 
+### zeebe.broker.gateway.multitenancy
```

Comment:
:x: I'm not sure adding only the multi-tenancy-related Gateway properties to the Broker config docs is a good idea.

For an embedded gateway setup, any Gateway property can be passed along to the `StandaloneBroker` by prefixing `zeebe.broker`. It might be confusing to users if they see some Gateway-related properties in this docs section, but not all.

This is a general problem that we're facing with Gateway-related properties in the Broker config docs.

:thinking: Maybe we can create a separate section for "Multi-tenancy configuration" (only a name suggestion) similar to the "Experimental configuration" at the end. In the "Multi-tenancy configuration" section we can link to the Gateway config docs related to MT, or move the new text that was added.

Reply:
I like your suggestion. I will add that to this PR and see if that gives us a temporary solution to move forward.